src/hardware/addon-protocols/rs232-usb.md

```diff
@@ -4,37 +4,232 @@ permalink: /hardware/addon-protocols/rs232-usb/
 title: Add-On Protocol - RS232 & USB
 ---
 
-External devices can communicate with the Geotab GO device through the revised Third-Party Data protocol. An initial Handshake will need to be successfully negotiated in order for the GO device to accept third-party data. Data can only be sent in a predefined format and will be saved and sent to MyGeotab.
+External devices can communicate with the Geotab GO device through the Third-Party RS232 and USB protocol below. The hardware interface will be one of the following:
 
-## Special Requirements for USB Integrations
+ - [IOX-RS232 F/M](https://www.geotab.com/documentation/iox-rs232/ "IOX-RS232 Support Documentation")
+ - [IOX-USB](https://www.geotab.com/documentation/iox-usb/ "IOX-USB Support Documentation")
 
-The IOX-USB operates as a USB 2.0 full-speed host. The maximum data transfer rate is 12 Mbit/s. The IOX-USB can use two methods to enumerate a USB device:
-
-1. The [Android Open Accessory protocol](https://source.android.com/devices/accessories/protocol.html). This [sample project](https://github.com/Geotab/android-external-device-example) can be used as a framework.
-
-2. USB-CDC (Communications Device Class)
-
-### Enabling IOX-USB Data Transfer
+### Special Requirements
+##### Enabling IOX-USB Data Transfer
 
 To enable third-party data communication on the IOX-USB, apply the following custom parameter to the GO device through MyGeotab.
 
 ```xml
 <GoParameters><Parameter Description="Enable USB Data" Offset="164" Bytes="02"/></GoParameters>
 ```
 
-The GO device will automatically upgrade to the ProPlus plan — with all associated charges — once third-party data transfer begins.
+The GO device will automatically upgrade to the ProPlus rate plan once third-party data transfer begins.
+##### IOX-USB Communication Consideration
+ 
+The IOX-USB operates as a USB 2.0 full-speed host. The maximum data transfer rate is 12 Mbit/s. The IOX-USB can use two methods to enumerate a USB device:
+
+- The [Android Open Accessory protocol (*AOA*)](https://source.android.com/devices/accessories/protocol.html). This [sample project](https://github.com/Geotab/android-external-device-example "Android Open Accessory Sample") can be used as a framework.
+- USB-CDC (Communications Device Class)
+
+##### Powering a device using the IOX-NFC
```

Comment:
IOX-NFC? That doesn't sound right.

Reply:
Definitely not right, should just say IOX. Looks like a typo.